src/devtoolsPanel.ts

```diff
@@ -365,6 +365,44 @@ export class DevToolsPanel {
             `;
     }
 
+    private getCdnHtmlForWebview() {
+        let cdnBaseUrl = this.config.devtoolsBaseUri;
+        if (!cdnBaseUrl) {
+            // Not provided, calculate based on config
+            // TODO: CDP call to determine actual hash and stuff
+            cdnBaseUrl = this.config.useLocalEdgeWatch ? 'http://localhost:3000/vscode_app.html' : 'https://devtools.invalid';
+        }
+        const hostPath = vscode.Uri.file(path.join(this.extensionPath, 'out', 'host_beta', 'host.bundle.js'));
+        const hostUri = this.panel.webview.asWebviewUri(hostPath);
+
+        const stylesPath = vscode.Uri.file(path.join(this.extensionPath, 'out', 'common', 'styles.css'));
+        const stylesUri = this.panel.webview.asWebviewUri(stylesPath);
+
+        // the added fields for "Content-Security-Policy" allow resource loading for other file types
+        return `
+            <!doctype html>
```

Comment:
we should add the styles to render the frame correctly, it is really small ATM
![image](https://user-images.githubusercontent.com/43078681/129073141-362b4df9-6140-4fd2-85f6-52caf662505b.png)

Reply:
So, I used to see this, but I no longer see this particular problem. If you continue to repro, let's call and see if we can figure out what is going on

Reply:
somehow it stopped after updating VSCode, probably has some kind of caching but it is now fixed :), thanks Brandon